CPP20.md

```diff
@@ -435,6 +436,23 @@ Provides a standard type for representing UTF-8 strings.
 char8_t utf8_str[] = u8"\u0123";
 ```
 
+### constinit specifier
+`constinit` specifier asserts that a variable has a constant or zero initialization and `constinit` can only be applied to variables with static storage duration.
+The initialization of a variable with static storage duration can take place in one of the following two cases:
+1. At compile time using either constant initialization or zero initialization.
+2. At runtime when the control first passes through its declaration.
+
+The second way of initialization may lead to some dangerous bugs because, C++ does not guarantee the order of static variable initialization at runtime which is generally referred as *static initialization order fiasco*.
+
+A variable with static storage duration if decorated with `constinit`, then the variable **must** be initialized at compile-time. If not the program will give a compiler error.
```

Comment:
I think the following is more grammatically correct (I also removed the bolded text):

If a variable with static storage duration is decorated with `constinit`, then it must be initialized at compile-time, otherwise the program is ill-formed.